Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -454,6 +455,7 @@ CClientGame::~CClientGame(void)
     pKeyBinds->SetAllControlsEnabled(true, true, true);
     g_pCore->ForceCursorVisible(false);
     SetCursorEventsEnabled(false);
+    g_pMultiplayer->SetTakePedAStepHandler(NULL);
```

Comment:
Please use **nullptr** and move the line up to the other `g_pMultiplayer->...` lines